pkg/controllers/user/alert/configsyncer/configsyncer.go

```diff
@@ -368,13 +393,12 @@ func (d *ConfigSyncer) newRoute(match map[string]string, groupWait, groupInterva
 
 	gw := model.Duration(time.Duration(groupWait) * time.Second)
 	route.GroupWait = &gw
+
 	ri := model.Duration(time.Duration(repeatInterval) * time.Second)
 	route.RepeatInterval = &ri
 
-	if groupInterval != defaultGroupInterval {
-		gi := model.Duration(time.Duration(groupInterval) * time.Second)
-		route.GroupInterval = &gi
-	}
+	gi := model.Duration(time.Duration(groupInterval) * time.Second)
```

Comment:
Don't we need default interval? Or we just move it to somewhere else.

Reply:
the default value is configured in types, this is for event, because event will disappear quickly, so we use a shorter time.